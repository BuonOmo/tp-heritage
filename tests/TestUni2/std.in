S s1 0 3 2 50
S s2 15 3 4 5
PC p1 56 46 108 4536 80 100
R r1 12 12 50 200
OR union s1 s2
OR bigunion union r1 p1 s1
LIST
EXIT
