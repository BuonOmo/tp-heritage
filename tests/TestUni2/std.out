OK
OK
OK
OK
OK
OK
OR bigunion bigunion_union bigunion_r1 bigunion_p1 bigunion_s1
4
OR bigunion_union bigunion_union_s1 bigunion_union_s2
2
S bigunion_union_s1 0 3 2 50
S bigunion_union_s2 15 3 4 5
R bigunion_r1 12 12 50 200
PC bigunion_p1 56 46 108 4536 80 100
S bigunion_s1 0 3 2 50
PC p1 56 46 108 4536 80 100
R r1 12 12 50 200
S s1 0 3 2 50
S s2 15 3 4 5
OR union union_s1 union_s2
2
S union_s1 0 3 2 50
S union_s2 15 3 4 5
